### PR TITLE
main: Add offline support for check, pull, install and update commands

### PR DESCRIPTION
**Pull request by detsch**

### CODE REVIEW

**Review by mike-sul — APPROVED** [review]
LGTM, will do e2e testing tomorrow.

File: src/main.cc [hunk 1]
```diff
@@ -125,6 +143,15 @@ static int install(LiteClient& client, const bpo::variables_map& params, aklite:
     install_mode = params.at("install-mode").as<std::string>();
   }
 
+  LocalUpdateSource local_update_source;
```
Comment (mike-sul) [inline src/main.cc:146]: It seems like there is missing change for the pull command. As result the pull and install doesn't work (the last one is supposed to work but it depends on pull which doesn't work).

Comment (detsch) [reply src/main.cc:146]: You are right. I've fixed it now.

**Comment by mike-sul** [comment]
@detsch We need to make clang tidy happy https://github.com/foundriesio/aktualizr-lite/actions/runs/10565675394/job/29270730335?pr=352

**Pull request merged**